src/nwp_consumer/internal/inputs/ecmwf/mars.py

```diff
@@ -263,7 +263,7 @@ def mapTemp(self, *, p: pathlib.Path) -> xr.Dataset:  # noqa: D102
         wholesaleDataset = wholesaleDataset \
             .rename({"time": "init_time"}) \
             .expand_dims("init_time") \
-            .to_array(dim="variable", name="UKV") \
+            .to_array(dim="variable", name=f"ECMWF_{self.area}".upper()) \
```

Comment:
I would say just having it as ECMWF would be fine, or IFS if its from the IFS model, or ECMWF_Ensemble if its from the ensemble models, the area then somewhat makes it more complicated to load later. Having it as the model name then allows us to know which model made the data, if we combine multiple or something. This is fine leaving it with the area in the name, but I think it might be better if its just the model name, and not area. UKV has the area in the name, as its the UK high-resolution model, but ECMWF would be the same globally.

Reply:
Essentially, in my head, if we combine multiple NWP data sources into a single Xarray Dataset, the names should tell the model that is used, so [UKV, ECMWF, ICON-Global, ICON-EU, Arpege-World, etc], and we can figure out the area from the coordinates, but get the model name from the name of the data arrays. But open to other suggestions!